voxel51/platform/api.py

```diff
@@ -78,6 +79,38 @@ def close(self):
         if self.keep_alive:
             self._requests.close()
 
+    def get_job_url(self, job_id, url_type):
```

Comment:
For clarity, can you define `get_job_*_url` methods for `* = status, log, output`?

If you want to call this method internally as `_get_job_url` that's fine, but I'd like the public-facing API of this class to be as clear as possible.